packages/manager-components/src/components/navigation/menus/action/action.spec.tsx

```diff
@@ -0,0 +1,33 @@
+import { waitFor, screen } from '@testing-library/react';
+import { actionMenu } from './action.stories';
+import { ActionMenu, ActionMenuProps } from './action.component';
+import { render } from '../../../../utils/test.provider';
+import '@testing-library/jest-dom';
+
+const setupSpecTest = async (customProps?: Partial<ActionMenuProps>) =>
+  waitFor(() => render(<ActionMenu {...actionMenu} {...customProps} />));
+
+describe('ActionMenu', () => {
+  it('renders menu actions correctly', async () => {
+    const { container } = await setupSpecTest();
+
+    const action1 = screen.getByText('Action 1');
+    const action2 = screen.getByText('Action 2');
+    expect(action1).toBeInTheDocument();
+    expect(action2).toBeInTheDocument();
+    expect(
+      container.children[0].children[0].children[0].children[0].getAttribute(
+        'name',
+      ),
+    ).toBe('arrow-down-concept');
+  });
+
+  it('renders compact menu correctly', async () => {
+    let { container } = await setupSpecTest({ compact: true });
```

Comment:
**nitpick:** Can you take care of the lint errors?
```suggestion
    const { container } = await setupSpecTest({ compact: true });
```

Reply:
Done